src/random_test.cpp

```diff
@@ -258,6 +260,15 @@ int rand_int(int upper, int lower) {
   return dist(rng);
 }
 
+/* return random float number in the range of upper and lower */
+std::string rand_float(float upper, float lower, int precision) {
+  std::uniform_real_distribution<> dis(lower, upper);
+  std::ostringstream out;
+  out << std::fixed;
+  out << std::setprecision(precision) << (float)(dis(rng)*100.0)/100.0;
```

Comment:
(float)(dis(rng)*100.0)/100.0. 
can it be done with dis(rng)

Reply:
Fixed